elastic/kibana
# [Synthetics] fix local params in add monitor and edit monitor

## Summary

Resolves https://github.com/elastic/kibana/issues/150599
Resolves https://github.com/elastic/kibana/issues/150597

Allows specifying local params on the add monitor page.

<img width="1229" alt="Screen Shot 2023-02-08 at 11 43 38 AM" src="https://user-images.githubusercontent.com/11356435/217596642-2debe5df-17af-4d7f-8a05-12325e8c9420.png">
<img width="1497" alt="Screen Shot 2023-02-08 at 11 44 52 AM" src="https://user-images.githubusercontent.com/11356435/217596636-a2795cbd-e1cd-4ec4-a384-33362758589a.png">

Also enables editing params on the the edit monitor page.
<img width="1243" alt="Screen Shot 2023-02-08 at 11 46 09 AM" src="https://user-images.githubusercontent.com/11356435/217596729-009506d6-2bed-4f46-a81a-f9c7856ebbd8.png">


### Testing

1. Create a browser monitor that relies on a local param
2. Ensure the monitor runs as expected with the local param
3. Edit the monitor. Ensure the local params are visible in the edit monitor flow.
